ui/admin/view.php

```diff
@@ -182,24 +180,18 @@
 				}//end if
 
 				if ( 0 < count( $fields ) ) {
+					if ( $more ) {
+						$title = __( 'More Fields', 'pods' );
+					} else {
+						$title = __( 'Fields', 'pods' );
+					}
+
+					/** This filter is documented in classes/PodsMeta.php */
+					$title = apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: f03c340e77ab6edf9e2cc1bc991bce0a</sub>